core/common/src/main/java/alluxio/Constants.java

```diff
@@ -105,6 +105,8 @@
   public static final String KEY_VALUE_MASTER_CLIENT_SERVICE_NAME = "KeyValueMasterClient";
   public static final String KEY_VALUE_WORKER_CLIENT_SERVICE_NAME = "KeyValueWorkerClient";
 
+  public static final int DEFAULT_REGISTRY_GET_TIMEOUT_MS = 60 * SECOND_MS;
```

Comment:
maybe call this `WORKER_CREATION_TIMEOUT_MS` to make it clearer what it means

Reply:
it's the timeout of getting a class from the registry, not for the worker creation.

Reply:
ack